jdxsmith/rancid-tomatillos
# Refactor/api call

#### What’s this PR do?
* Unit & integration tests are written & passing
* Asynchronous testing is completed

#### Where should the reviewer start?
* `src/App.js` line 5
* `src/App.test.js` line 1
* `src/Movies.js` line 11
* `src/apiCalls.js` line 1

#### How should this be manually tested?
* Pull down and first note, when `npm start` is run, that the app is rendered
* Second, run `npm test` and note that all the tests are passing

#### Any background context you want to provide?
* Heavy on react, jest, and async 

#### What are the relevant tickets?
*  Closes #11 
* Closes #12 

#### Questions:
* What is the proper way to handle a 500 error?
